honeybee_radiance/cli/view.py

```diff
@@ -191,7 +195,12 @@ def merge_view(input_folder, base_name, extension, scale_factor, folder, name):
 
         # search for a single .vf in the folder and, if it's found, grab the info
         views = sorted(f for f in os.listdir(input_folder) if f.endswith('.vf'))
-        if len(views) == 1:  # replace the header with the info in the view
+        if view:
+            view_obj = View.from_file(view)
+            getinfo = Getinfo(output=output_file)
+            getinfo.options.a = 'VIEW= {}'.format(view_obj)
+            pfilt.pipe_to = getinfo
+        elif len(views) == 1:  # replace the header with the info in the view
             view_obj = View.from_file(os.path.join(input_folder, views[0]))
             getinfo = Getinfo(output=output_file)
             getinfo.options.a = 'VIEW= {}'.format(view_obj)
```

Comment:
What if there are multiple views and the original view is not provided. Can we do the quick math as suggested by @chriswmackey to re-create the original view?

Reply:
I believe if there are multiple views, we will assume that those are subviews. Then we can do the math, basically the reverse of the [grid](https://github.com/ladybug-tools/honeybee-radiance/blob/0de8d9a5ee99332b9046c133e488e9846f0f3ba1/honeybee_radiance/view.py#L632) method that I also mentioned.
If there are multiple views and they are **not** subviews, e.g. different view point or directions, then the reverse grid method will fail.